presto-parser/src/main/java/io/prestosql/sql/SqlFormatter.java

```diff
@@ -1219,6 +1220,28 @@ protected Void visitInsert(Insert node, Integer indent)
             return null;
         }
 
+        @Override
+        protected Void visitUpdate(Update node, Integer indent)
+        {
+            builder.append("UPDATE ")
+                    .append(node.getTable().getName());
+
+            builder.append("\n")
+                    .append(indentString(indent + 1))
+                    .append("SET ");
+            builder.append(node.getAssignments().stream()
+                    .map(assignment -> format("%s = %s", assignment.getName().getValue(), formatExpression(assignment.getValue())))
```

Comment:
Let's put each one on a separate line, so it looks like
```sql
UPDATE foo SET
    abc = 123,
    xyz = 456
WHERE something

Reply:
Changed.